app/apps/camunda/views.py

```diff
@@ -30,11 +33,23 @@ def complete_task(self, request):
         serializer = CamundaTaskCompleteSerializer(data=request.data)
 
         if serializer.is_valid():
-            camunda_response = CamundaService().complete_task(
-                serializer.data["camunda_task_id"], serializer.data["variables"]
-            )
+            case_id = serializer.data["case"]
+            case = Case.objects.get(id=case_id)
+            task_id = serializer.data["camunda_task_id"]
+            variables = serializer.data["variables"]
+            author = request.user
+            task = CamundaService().get_task(task_id)
+            task_name = task["name"]
```

Comment:
Dezelfde comment hier dat we users toevoegen dat we weten wie de taak heeft uitgevoerd ook in camunda